### PR TITLE
Assigned instanceID as the test name for e2e jobs

### PR DESCRIPTION
**Pull request by panktishah26**
*Issue #, if available:* [#306](https://github.com/aws/eks-anywhere/issues/306)

*Description of changes:*
For tests running in the e2e job, assign instanceID as the test name.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/338#" title="Author self-approved">panktishah26</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere/blob/main/OWNERS)~~ [panktishah26]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by mrajashree — APPROVED** [review]
/lgtm

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/338#pullrequestreview-766947852" title="LGTM">mrajashree</a>*, *<a href="https://github.com/aws/eks-anywhere/pull/338#" title="Author self-approved">panktishah26</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere/blob/main/OWNERS)~~ [mrajashree,panktishah26]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by mrajashree** [comment]
oh hold till we modify the cleanup job

**Comment by abhay-krishna** [comment]
@panktishah26: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
eks-anywhere-mocks-presubmits | a6d41393fd8bdd8f32889c48c45a4f6fc17044ae | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere/338/eks-anywhere-mocks-presubmits/1443348039774244864) | `/test eks-anywhere-mocks-presubmits`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere&pr=338). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:panktishah26).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by abhay-krishna** [comment]
@panktishah26: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
eks-anywhere-mocks-presubmits | a6d41393fd8bdd8f32889c48c45a4f6fc17044ae | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere/338/eks-anywhere-mocks-presubmits/1443348039774244864) | `/test eks-anywhere-mocks-presubmits`
eks-anywhere-presubmit | a6d41393fd8bdd8f32889c48c45a4f6fc17044ae | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere/338/eks-anywhere-presubmit/1443348039803604992) | `/test eks-anywhere-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere&pr=338). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:panktishah26).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by abhay-krishna** [comment]
@panktishah26: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
eks-anywhere-mocks-presubmits | a6d41393fd8bdd8f32889c48c45a4f6fc17044ae | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere/338/eks-anywhere-mocks-presubmits/1443348039774244864) | `/test eks-anywhere-mocks-presubmits`
eks-anywhere-presubmit | a6d41393fd8bdd8f32889c48c45a4f6fc17044ae | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere/338/eks-anywhere-presubmit/1443348039803604992) | `/test eks-anywhere-presubmit`
eks-anywhere-e2e-presubmit | a6d41393fd8bdd8f32889c48c45a4f6fc17044ae | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere/338/eks-anywhere-e2e-presubmit/1443348039740690432) | `/test eks-anywhere-e2e-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere&pr=338). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:panktishah26).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by abhay-krishna** [comment]
@panktishah26: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
eks-anywhere-mocks-presubmits | a6d41393fd8bdd8f32889c48c45a4f6fc17044ae | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere/338/eks-anywhere-mocks-presubmits/1443348039774244864) | `/test eks-anywhere-mocks-presubmits`
eks-anywhere-presubmit | a6d41393fd8bdd8f32889c48c45a4f6fc17044ae | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere/338/eks-anywhere-presubmit/1443348039803604992) | `/test eks-anywhere-presubmit`
eks-anywhere-e2e-presubmit | a6d41393fd8bdd8f32889c48c45a4f6fc17044ae | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere/338/eks-anywhere-e2e-presubmit/1443348039740690432) | `/test eks-anywhere-e2e-presubmit`
eks-anywhere-attribution-files-presubmit | a6d41393fd8bdd8f32889c48c45a4f6fc17044ae | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere/338/eks-anywhere-attribution-files-presubmit/1443348039702941696) | `/test eks-anywhere-attribution-files-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere&pr=338). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:panktishah26).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Review by mrajashree — APPROVED** [review]
/lgtm

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/338#pullrequestreview-768233596" title="LGTM">mrajashree</a>*, *<a href="https://github.com/aws/eks-anywhere/pull/338#" title="Author self-approved">panktishah26</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere/blob/main/OWNERS)~~ [mrajashree,panktishah26]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
